protocol/socketio.go

```diff
@@ -125,16 +125,19 @@ func getAck(text string) (ackId int, restText string, err error) {
 	text = text[2:]
 
 	pos := strings.IndexByte(text, '[')
+	payloadIdx := pos
 	if pos == -1 {
 		return 0, "", ErrorWrongPacket
 	}
-
+	if p := strings.IndexByte(text[0:pos], ','); p >=0 {
```

Comment:
we forced introduced a colon (`,`) in the Encode function but this change was not handled here and therefore this function was not working as expected